docs/ocis/development/beta-testplan.md

```diff
@@ -0,0 +1,293 @@
+| create & share a space with a group with editor role, do CRUD file/folder operations with WebDAV                  | space and content should be writable                                              |                   |
+| create a space, try CRUD file/folder operations with WebDAV on the space with a user that its not shared with     | space and content should not be accessible                                        |                   |
+| create a space with a quota, share the space, upload files till the quota is exceeded                             | upload should work till quota is full, uploads should not work when quota is full |                   |
+| share file/folders from inside a space (see other sharing section)                                                | sharing works and obeys the permissions                                           |                   |
+| create a space, rename the space                                                                                  | new name should be displayed in API calls and web                                 |                   |
+| create a space, change description of the space                                                                   | new description should be displayed in API calls and web                          |                   |
+| create a space, set quota, change quota of the space                                                              | new quota is obeyed                                                               |                   |
+| create a space, set quota, change quota of the space to a value that is lower than the sum of data already stored | new quota is obeyed, new files cannot be uploaded                                 |                   |
+| try the various space operations with invalid data                                                                | good error output, server does not crash                                          |                   |
+| try the various space operations without the correct permissions                                                  | operations are not possible without sufficient permissions                        |                   |
+| try the various space operations on personal and virtual spaces                                                   | server should not crash, good error responses                                     |                   |
+| try the various space operations sending invalid data (invalid numbers, wrong types, invalid JSON, etc.)          | server should not crash, good error responses                                     |                   |
+
+
+## Web
+
+Prerequisite:
+- [connect ocis to your preferred LDAP server](https://owncloud.dev/ocis/deployment/ocis_ldap/)
+- create users and groups in LDAP
+- Use your preferred browser
```

Comment:
which browsers should we use? do we have a list (maybe versions)?

Reply:
added some browsers from https://github.com/owncloud/web/blob/92199f740266b7c945ef956525ff90a8db5df018/package.json#L30
Not strictly all, so not have to differentiate between mobile and desktop here, but a subset